test/help.cc

```diff
@@ -17,14 +17,35 @@ TEST(FlagsTest, Help) {
   int argc = arguments.size();
   const char** argv = arguments.data();
 
-  EXPECT_EXIT(
-      parser.Parse(&argc, &argv),
-      testing::ExitedWithCode(0),
+// On POSIX systems (e.g. Linux, Cygwin, and Mac), googletest
```

Comment:
Wow that's super annoying huh!

I see there is a quote here that says "If you want your death tests to work in all cases, you can either #if on these macros or use the more limited syntax only." so rather than have what almost looks like the same regular expression let's just always use the simple regular expression implementation!

That way you won't have to use any `#ifdef`

Reply:
Yep, great catch!